Automattic/jetpack
# Remove subscriptions newsletter feature flag

Closes: 105-gh-Automattic/gold

## Proposed changes:

* Remove the `jetpack_subscriptions_newsletter_feature_enabled` filter that gated access to a feature while in development. This feature flag is no longer needed.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* On an installation of WordPress, make sure the Jetpack Beta plugin is installed, and then activate this branch.
* Check this patch out to sun on sandbox.
* Ensure subscribe and public-api are pointed to sandbox.
* Go to [the earn settings page](https://wordpress.com/earn/payments/) and ensure Stripe is setup in test mode.
* Create a new page or post and add the subscribe block.
* Publish the page and try subscribing. Confirm the block works as expected.
* Try testing the other earn blocks to ensure there isn't a regression.
